bench/bench.ml

```diff
@@ -98,9 +97,7 @@ let dune_build () =
   let stderr_to = Process.Io.make_stderr Swallow in
   let open Fiber.O in
   let+ times =
-    Process.run_with_times dune
-      ~display:!Dune_engine.Clflags.display
-      ~stdin_from ~stdout_to ~stderr_to
+    Process.run_with_times dune ~display:Quiet ~stdin_from ~stdout_to ~stderr_to
       [ "build"; "@install"; "--release" ]
```

Comment:
Here we run Dune and record timing data. We don't need to see this reported to the user.